copper-server-api/src/main/java/com/pogeyan/cmis/api/data/ITracingFacade.java

```diff
@@ -0,0 +1,15 @@
+package com.pogeyan.cmis.api.data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public interface ITracingFacade {
```

Comment:
Please add code documentation for this interface, and rename this as "ITracingService"